R/profilePlot.R

```diff
@@ -337,28 +273,29 @@ profilePlot <- function(contrastDF,
         if (sizeBySignificance) {
             var.annot <- contrastDF %>%
                 dplyr::select(Group, negLog10P)
-            showSizeLegend <- TRUE
             sizeBy <- "negLog10P"
+            showSizeLegend <- TRUE
         } else {
             var.annot <- contrastDF %>%
                 dplyr::select(Group)
-            showSizeLegend <- FALSE
             sizeBy  <- "Group"
+            showSizeLegend <- FALSE
         }
 
-        if (!missing(geneSymCol)) {
-            var.annot <- merge(var.annot, contrastDF %>%
-                                   dplyr::select(!!dplyr::sym(geneSymCol)),
-                               by = 0)
-            rownames(var.annot) <- var.annot$Row.names
-            var.annot$Row.names <- NULL
+        if (!missing(geneNameCol)) {
+            gene_data <- DGEobj::getItem(dgeObj, "geneData") %>%
+                dplyr::select(all_of(geneNameCol))
```

Comment:
- It is one column so why using `all_of` function.
- missing `dplyr::` before `all_of`

Reply:
It makes a waring message if we not using all_of()